### PR TITLE
Updated README file with examples of directories structures

### PR DESCRIPTION
**Pull request by zyuzka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,35 @@ If you only need to add a new Spryk configuration you will start by adding an In
...
+│   └─── ...
+│   └─── spryk
+│   │    └─── spryks
+│   │         │   ...
+│   │         │   spryk-name.yml
+│   │         │   ...
+│   └─── ...
+```
+
+If you selected the template Spryk you will most likely see the error that the defined template file could not be found. In this case you need to add your template to `config/spryk/templates` on project or core layer:
```
Comment (dereuromark) [inline README.md]: Missing comma after Spryk. Relative clauses started with if always need that.

Comment (dereuromark) [reply README.md]: `config/spryk/templates` with trailing / please.

File: README.md [hunk 2]
```diff
@@ -102,6 +102,35 @@ If you only need to add a new Spryk configuration you will start by adding an In
...
+
+```
+project OR package root directory
+│
+└─── config
+│   └─── ...
+│   └─── spryk
+│   │    └─── spryks
```
Comment (dereuromark) [inline README.md]: spryks/ etc would make it more clear those are dirs, not files. (Some files do not have ext).

File: README.md [hunk 3]
```diff
@@ -102,6 +102,35 @@ If you only need to add a new Spryk configuration you will start by adding an In
...
 
-You need to add your Spryk definition file into `vendor/spryker-sdk/spryk/config/spryk/spryks` when you added your new Spryk definition file re-run the tests. Now it dependends on the spryk type you choosed for your Spryk definition what comes next.
+You need to add your Spryk definition file into `config/spryk/spryks` on project or core layer:
 
-If you selected the template Spryk you will most likely see the error that the defined template file could not be found. In this case you need to add your template to `vendor/spryker-sdk/spryk/config/spryk/templates` when this is done re-run your tests. Now you should see a green test.
+```
+project OR package root directory
+│
+└─── config
+│   └─── ...
```
Comment (stereomon) [inline README.md]: What is this for? From the SprykConfig
```
$projectSprykDirectory = realpath($this->getRootDirectory() . 'config/spryk/' . $subDirectory);
$sprykModuleDirectory = realpath($this->getSprykCorePath() . 'config/spryk/' . $subDirectory);
```

It should be config/spryk or not?

File: README.md [hunk 4]
```diff
@@ -102,6 +102,32 @@ If you only need to add a new Spryk configuration you will start by adding an In
 
 When this is done run the Integration tests with `vendor/bin/codecept run Integration -g {YOUR TEST GROUP}` and see the test failing. You will get a message that the Spryk definition was not found by the given name, so add the definition file for you new Spryk. 
 
-You need to add your Spryk definition file into `vendor/spryker-sdk/spryk/config/spryk/spryks` when you added your new Spryk definition file re-run the tests. Now it dependends on the spryk type you choosed for your Spryk definition what comes next.
+You need to add your Spryk definition file into `config/spryk/spryks` on project or core layer:
```
Comment (dereuromark) [inline README.md]: layer or level?
I think we usually refer to the subnames as (application or otherwise) layers.

**Pull request merged**